gdcmodels/esutils.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import urllib3
+
+from time import sleep
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionTimeout
+
+UPDATE_INTERVAL = 10
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_elasticsearch(host, port=9200, user='', password='', use_ssl=True):
```

Comment:
Since we are pushing towards the library route, it might make sense to not assume how (what extra options are needed etc) the library users wants to connect to elastic-search and accept an appropriate es instance (as provided in this repo by config.es) or provide a much more robust/generic configuration option, like connecting to multiple nodes, authentication, ssl_context etc. I will vote for not exposing this as part of esutil and let users pass it to the exposed library functions. Since there is a script here also, you can make this more specific for that script by moving it to the script module